test/integration/tests/3396-package-indices/Main.hs

```diff
@@ -0,0 +1,13 @@
+import           StackTest
+import           System.Directory   (createDirectory)
+import           System.Environment (getEnv)
+import           System.FilePath    ((</>))
+
+main :: IO ()
+main = do
+  home <- getEnv "HOME"
+  createDirectory (home </> ".stack" </> "indices" </> "CustomIndex")
+  copy "CustomIndex/00-index.tar" (home </> ".stack"  </> "indices" </> "CustomIndex" </> "00-index.tar")
+  copy "CustomIndex/01-index.tar" (home </> ".stack"  </> "indices" </> "CustomIndex" </> "01-index.tar")
+  copy "CustomIndex/01-index.cache" (home </> ".stack"  </> "indices" </> "CustomIndex" </> "01-index.cache")
```

Comment:
Better not to copy over the `cache`, especially since the format will change over time. Also, I'd be interested in whether this works without the `00-index.tar` file as well.

Reply:
@snoyberg good point. Removing the cache file. Also, 00-index.tar is not needed so removing that as well.